io-engine/src/core/snapshot.rs

```diff
@@ -25,6 +26,35 @@ impl SnapshotParams {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct ListSnapshotParams {
```

Comment:
Could we rename 'ListSnapshotParams' to something like 'VolumeSnapshotDescriptor' or so ?
'Params' usually denote input parameters. :-)

Reply:
> Could we rename 'ListSnapshotParams' to something like 'VolumeSnapshotDescriptor' or so ? 'Params' usually denote input parameters. :-)

Accepted. Code changed.